### ISSUE TITLE
Exoplanet Explorer won't load due to bower.json issue

### ISSUE THREAD

**Issue by sarah-maris** [udacity/fend-issues#52]
Link to lesson:  https://classroom.udacity.com/nanodegrees/nd001/parts/91561162-9864-4caf-b2aa-e6504385e4e2/modules/550593026975460/lessons/5972243496/concepts/60561938060923

When bower install is run, this error appears:
> ECONFLICT Unable to find suitable version for test-fixture

and the bower install fails.

If I eliminate the "test-fixture" dependency  the bower install is successful and the app appears to run as expected (although I have not fully tested this ....)

**Comment by sarah-maris** [comment]
Two more students reporting this issue

**Comment by sarah-maris** [comment]
And two more this week.

**Comment by forbiddenvoid** [comment]
I'm working with the content developer who built this project on finding a solution. There have been a lot of updates to Polymer in the last two years.

**Comment by asparr** [comment]
@forbiddenvoid There is 1 pull request and 1 issue on the project in GitHub that you might ask him to check out as well: https://github.com/udacity/exoplanet-explorer/issues/6

**Comment by forbiddenvoid** [comment]
The version resolution issue appears to have been fixed by changes in the dependency tree. Please report this again if the error resurfaces.

**Comment by sarah-maris** [comment]
I am still getting this issue.  Can only run the project if the `"test-fixture": "PolymerElements/test-fixture#^1.0.0"` dependency is removed.

**Comment by sarah-maris** [comment]
Solution!  Run bower install in the Windows cmd shell instead of Git Bash.  You ill be prompted to choose an option:
```
Unable to find a suitable version for test-fixture, please choose one by typing one of the numbers below:
    1) test-fixture#^1.0.0 which resolved to 1.1.2 and is required by polymer-starter-kit
    2) test-fixture#^3.0.0 which resolved to 3.0.0 and is required by web-component-tester#6.5.0

Prefix the choice with ! to persist it to bower.json
```
Enter `2!`
